root-files/opt/flownative/lib/nginx-legacy.sh

```diff
@@ -184,7 +191,7 @@ EOM
     # pass persistent resource requests to GCS
     location ~* "^${BEACH_PERSISTENT_RESOURCES_BASE_PATH}([a-f0-9]{40})/" {
         resolver 8.8.8.8;
-        expires 3600;
+        expires ${NGINX_STATIC_FILES_LIFETIME};
```

Comment:
I'd not expect that `_STATIC_` variable to have an effect on `_Persistent_` resources…

Reply:
The "static" refers to files in the local file system, from the Nginx point of view (versus dynamic data, provided by PHP-FPM for example).

There's also a variable called "NGINX_STATIC_ROOT" which uses "static" for a similar purpose.